cgi/product_jqm_multilingual.pl

```diff
@@ -144,6 +144,9 @@ =head1 DESCRIPTION
 	print $out encode_json(Vars());
 	close $out;
 
+	# Processes fields to remove email values as entries
+	preprocess_product_field($product_ref);
```

Comment:
So as we discussed yesterday, I think it would be better to process each field when we input it: https://github.com/openfoodfacts/openfoodfacts-server/blob/main/cgi/product_multilingual.pl#L586

We have lines like this one:
$product_ref->{$field} = remove_tags_and_quote(decode utf8 => single_param($field));

This is the line that reads the value entered by the user, and removes HTML tags from it.

Instead we could do this:

$product_ref->{$field} = preprocess_product_field($field, decode utf8 => single_param($field));

The preprocess_product_field() would then call remove_tags_and_quotes() for all fields, and unless the field can contain emails (I think only "customer_service" and "other_information"), it also calls remove_email()

Note that the field name $field would be passed as the first argument: preprocess_product_field($field, $value)
and the function would return the preprocessed value.

Reply:
Done the required changes!